pkg/proxy/iptables/proxier_test.go

```diff
@@ -293,36 +270,22 @@ func TestDeleteEndpointConnections(t *testing.T) {
 			},
 		}
 
-		fp.deleteEndpointConnections(input)
-
-		// For UDP and SCTP connections, check the executed conntrack command
-		var expExecs int
+		// For UDP and SCTP connections, add the flow to the FakeConntrackClearer
 		if conntrack.IsClearConntrackNeeded(tc.protocol) {
-			isIPv6 := func(ip string) bool {
-				netIP := net.ParseIP(ip)
-				return netIP.To4() == nil
-			}
 			endpointIP := utilproxy.IPPart(tc.endpoint)
-			expectCommand := fmt.Sprintf("conntrack -D --orig-dst %s --dst-nat %s -p %s", tc.svcIP, endpointIP, strings.ToLower(string((tc.protocol))))
-			if isIPv6(endpointIP) {
-				expectCommand += " -f ipv6"
+			ct.Proto = tc.protocol
+			ct.OrigDstIP = tc.svcIP
+			ct.OrigDstPort = int(tc.svcPort)
+			ct.NatDstIP = endpointIP
+			if tc.simulatedErr != "" && tc.simulatedErr != conntracktest.NoConnectionToDelete {
+				klog.Errorf(tc.simulatedErr)
 			}
-			actualCommand := strings.Join(fcmd.CombinedOutputLog[fexec.CommandCalls-1], " ")
-			if actualCommand != expectCommand {
```

Comment:
I have to come back to this test, I'm not getting exactly how this is testing if the conntrack gets deleted